src/main/java/io/github/moulberry/notenoughupdates/mixins/MixinOptifineCustomItems.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2023 NotEnoughUpdates contributors
+ *
+ * This file is part of NotEnoughUpdates.
+ *
+ * NotEnoughUpdates is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * NotEnoughUpdates is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.moulberry.notenoughupdates.mixins;
+
+import io.github.moulberry.notenoughupdates.NotEnoughUpdates;
+import io.github.moulberry.notenoughupdates.miscfeatures.StorageManager;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.inventory.GuiChest;
+import net.minecraft.item.ItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Pseudo;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Pseudo
+@Mixin(targets = "net.optifine.CustomItems", remap = false)
+public class MixinOptifineCustomItems {
+	@Inject(method = "getCustomItemProperties", at = @At(value = "HEAD"), cancellable = true)
+	private static void getCustomItemProperties(ItemStack itemStack, int type, CallbackInfoReturnable<?> cir) {
+		if (NotEnoughUpdates.INSTANCE.config.storageGUI.disableCIT &&	StorageManager.getInstance().isStorageOpen &&
+			Minecraft.getMinecraft().currentScreen instanceof GuiChest) cir.cancel();
```

Comment:
I'm not sure if i like using .cancel() here. That seems a bit confusing for a method that returns something (I assume this is meant to be the same as `cir.setReturnValue(null)`?

Reply:
I can't set return value because the method uses a custom class

Reply:
someone should stop me from making comments at 7am